edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/factory/ClauseFeatureExtractor.java

```diff
@@ -45,7 +45,14 @@ public ClauseFeatureExtractor(String pseudoParseViewName, String clauseViewName)
     @Override
     public Set<Feature> getFeatures(Constituent c) throws EdisonException {
         TextAnnotation ta = c.getTextAnnotation();
-        TreeView pseudoParseView = (TreeView) ta.getView(parseViewName);
+        TreeView pseudoParseView;
+        // If the real parse view exists, use that.
+        // Normally, this feature requires the PSEUDO_PARSE to be manually created by the user
+        // as it is cheaper to make than a full parse. However, there are cases
+        // where the user has already created a full parse view (for other features)
```

Comment:
I'm confused now; I thought we build pseudo-parse automatically based on other annotations (e.g. real parse): https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/0965deae01ff6bcd176f96b9fd050369f0ae3e1e/edison/src/main/java/edu/illinois/cs/cogcomp/edison/annotators/PseudoParse.java